src/test/regress/expected/citus_local_tables.out

```diff
@@ -295,18 +295,18 @@ SELECT create_reference_table('reference_table');
 
 -- show that colociation of citus local tables are not supported for now
 -- between citus local tables
-SELECT mark_tables_colocated('citus_local_table_1', ARRAY['citus_local_table_2']);
-ERROR:  citus local tables cannot be colocated with other tables
+SELECT update_distributed_table_colocation('citus_local_table_1', colocate_with => 'citus_local_table_2');
+ERROR:  relation citus_local_table_1 should be a hash distributed table
```

Comment:
Which error message do you think is better? The current one or the previous one.

The previous one says what the problem is, but doesn't provide what works for this UDF.
The current one says what the tables needs to be, but it doesn't directly say what the problem is.

Reply:
yes, the reason is `update_distributed_table_colocation` checks if the table is hash distributed or not first; while `mark_tables_colocated` checks if it's a citus local table or not first. shall we consider revising this check order?